networkx/generators/small.py

```diff
@@ -64,36 +64,51 @@ def LCF_graph(n, shift_list, repeats, create_using=None):
     """
     Return the cubic graph specified in LCF notation.
 
-    LCF notation (LCF=Lederberg-Coxeter-Fruchte) is a compressed
+    LCF (Lederberg-Coxeter-Fruchte) notation[1]_ is a compressed
     notation used in the generation of various cubic Hamiltonian
-    graphs of high symmetry. See, for example, dodecahedral_graph,
-    desargues_graph, heawood_graph and pappus_graph below.
+    graphs of high symmetry. See, for example, `dodecahedral_graph`,
+    `desargues_graph`, `heawood_graph` and `pappus_graph`.
 
-    n (number of nodes)
-      The starting graph is the n-cycle with nodes 0,...,n-1.
-      (The null graph is returned if n < 0.)
+    For each node ``n_i``, cycling through the n-cycle a total of ``len(shift_list) * repeats``
+    with shift cycling through `shift_list` `repeat` s times connects ``n_i``
+    with ``n_i + shift mod n``.
```

Comment:
This sentence is not clear to me -- though some of that may be because words and notation are defined in the parameters section below.  Is there a way to describe what this will say before saying it? Or maybe turning the sentence around with "Each node ``n_i`` connects with ``n_i + shift mod n`` where shift is cycling through input ``shift_list`` ... "... 

It would probably also help to say that all the nodes are the first N integers, with connections determined by relationships mod N. But I don't understand enough to describe that relationship yet. I understand much better than before though. :)

Reply:
I'm in the same boat - this was my feeble attempt at mashing together the existing description with what I could understand from reading the code. Let me take another stab incorporating these suggestions!

Reply:
I took your advice of flipping the sentence around which I agree improves the clarity. Whether or not it improves the situation *enough* is another question --- LMK what you think!